### ISSUE TITLE
Logs analysis : convert n_iter to n_epoch

### ISSUE THREAD

**Issue by joihn** [open-mmlab/mmdetection#6275]
in order to make my logs plot more intuituve, I want to put the `n_epoch` instead of `n_iter` on the X axis. 
I tried the folowing formula
`N_epoch = N_iter*batch_size / dataset_train_size`

But it doesn't yield the correct results. Any Idea why ?

(not that I'm using a validation set and that my workflow is the following :` workflow = [('train', 1),('val', 1)]` )
